# TrafficEventList Message

std_msgs/Header header

openpilot/TrafficEvent[] trafficEvents
